### PR TITLE
Build from src fix for jenkins

### PR DESCRIPTION
**Pull request by shapovalovdev**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -146,16 +152,18 @@ else
...
-# In case if contrail folder is not bint to the container from host
-# it is needed to copy container builde on host to be able mount
-# data into the building conainers for build from sources.
-if [[ -n "$CONTRAIL_BUILD_FROM_SOURCE" && "$BIND_CONTRAIL_DIR" == 'false' ]] ; then
-  if [[ ! -e ${CONTRAIL_DIR}/contrail-container-builder ]] ; then
-    sudo docker cp $TF_DEVENV_CONTAINER_NAME:/root/contrail/contrail-container-builder ${CONTRAIL_DIR}/
+# In case if contrail folder is not bound to the container from host
+# it is needed to copy contrail sources and container builder on host to be able mount
+# data into the building containers for build from sources.
+CONTRAIL_SRC_VOLUME=$(docker volume ls | grep "${SRC_VOLUME_NAME}")
```
Comment (alexey-mr) [inline run.sh]: SRC_VOLUME_NAME is empty for non src build case, so this grep always return something

Comment (alexey-mr) [inline run.sh]: if it is just a local variable please use lower case name

File: run.sh [hunk 2]
```diff
@@ -111,6 +116,11 @@ if ! is_container_created "$TF_DEVENV_CONTAINER_NAME"; then
   volumes+=" -v ${scriptdir}:/root/tf-dev-env:z"
   if [[ "$BIND_CONTRAIL_DIR" != 'false' ]] ; then
     volumes+=" -v ${CONTRAIL_DIR}:/root/contrail:z"
+  else
+    if [[ -n "$CONTRAIL_BUILD_FROM_SOURCE" && -n ${src_volume_name} ]] ; then      
+      docker volume create --name ${src_volume_name}
```
Comment (alexey-mr) [inline run.sh]: you dont need to create volume. it be created automatically on docker run if needed

Comment (shapovalovdev) [reply run.sh]: done

**Pull request merged**